test/e2e/flowaggregator_test.go

```diff
@@ -226,7 +227,22 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 		if err != nil || rc != 0 {
 			return false, err
 		}
-		return strings.Count(collectorOutput, srcIP) >= expectedNumDataRecords && strings.Count(collectorOutput, dstIP) >= expectedNumDataRecords, nil
+		if testOptions.providerName != "kind" {
+			recordSlices := getRecordsFromOutput(collectorOutput)
+			for _, record := range recordSlices {
+				exportTime := int64(getUnit64FieldFromRecord(t, record, "flowEndSeconds"))
+				if strings.Contains(record, srcIP) && strings.Contains(record, dstIP) {
+					if exportTime >= timeStartSec+iperfTime {
+						return true, nil
+					}
+					// t.Logf("considered flow export at %v, required >= %v + %v", exportTime, timeStartSec, iperfTime)
+				}
+			}
+			return false, nil
+		} else {
+			// Timestamp field is missing in dump-conntrack output on Kind cluster
```

Comment:
is timestamp field not maintained in OVS userspace datapath like counters or is there some other reason?

Reply:
Yes I think so, although there is code https://github.com/openvswitch/ovs/blob/f8be30acf2eb60d567bb7386b98f5cb58ddb9119/lib/ct-dpif.c#L299 to print timestamp field, but there is no implementation related to fetch these values https://github.com/openvswitch/ovs/blob/721488d4a5f497f5693e286cf6f1916804bce2eb/lib/conntrack.c#L2488. In kind cluster, I didn't find both start and stop timestamp in dump-conntrack command output. On Windows, I could find `start=` timestamp field only, which is related to https://github.com/openvswitch/ovs/blob/76b720ed1145560a4aaa394cde8cac3f718f40fd/datapath-windows/ovsext/Conntrack.c#L1761 and we mentioned in the PR before: https://github.com/antrea-io/antrea/blob/3134f7ee5471743e5dabe8c6416bd8d58a79a423/pkg/agent/flowexporter/connections/conntrack_ovs.go#L219

Reply:
Thanks for checking. A couple of things here:
1. We should raise this along with the counters issue, especially start timestamp is useful.
3.  Can we use the export time field here: `Exported Time: 1608338076 (2020-12-19 00:34:36 +0000 UTC)` instead? It should be available all the time and will not need any condition.

Reply:
Thanks, just found `stoptime` have been added in your recent PR (https://github.com/antrea-io/antrea/blob/3134f7ee5471743e5dabe8c6416bd8d58a79a423/pkg/agent/flowexporter/connections/conntrack_ovs.go#L269) for userspace datapath, we could use `flowEndSeconds` in kind cluster too.
Yes, start timestamp issue should be raised to OVS team together with counters, have you started the conversation with Ansis or William?

Reply:
Ok. I thought that was a reasonable thing to do if the `stopTime` field is not available in dumped connection. It may not be accurate for connections to be deleted. `flowEndSeconds` should be good.

I thought we had to go through Niaz.. Let us take this offline.